modules/ossm-cr-version.adoc

```diff
@@ -5,4 +5,4 @@
 [id="ossm-cr-version_{context}"]
 = version parameter
 
-You use the `version` parameter to specify what version of the control plane to install. When you create a `ServiceMeshControlPlane` object with an empty `version` parameter, the admission webhook sets the version to the current version. New `ServiceMeshControlPlanes` objects with an empty `version` parameter are set to `v2.0`. Existing `ServiceMeshControlPlanes` objects with an empty `version` parameter keep their setting.
+The {SMProductName} Operator supports installation of different versions of the `ServiceMeshControlPlane`. You use the version parameter to specify what version of the control plane to install. If you do not specify a version parameter when creating your SMCP, the Operator will apply the latest version: ({MaistraVersion}). Existing `ServiceMeshControlPlane` objects keep their version setting during upgrades of the Operator.  
```

Comment:
```suggestion
With the {SMProductName} Operator, you can install a specific version of the `ServiceMeshControlPlane` (SMCP). To specify the version, set the `version` parameter before you create the SMCP. Otherwise, the Operator applies the latest version, {MaistraVersion}. After upgrading the Operator, `ServiceMeshControlPlane` objects that have a specified version continue using that version.
```

Reply:
Do users need to know how to format the version number? If so, I'd add a sentence showing the format syntax between "SMCP." and "Otherwise."

Reply:
Please add a `:_content-type: <TYPE>` attribute to this file.

Reply:
`:_content-type: REFERENCE`?